lib/spack/spack/util/web.py

```diff
@@ -85,6 +85,21 @@ def handle_starttag(self, tag, attrs):
                     self.links.append(val)
 
 
+class IncludeFragmentParser(HTMLParser):
+    """This parser takes an HTML page and selects the include-fragments,
+    used on GitHub, https://github.github.io/include-fragment-element."""
+
+    def __init__(self):
+        HTMLParser.__init__(self)
```

Comment:
Why calling this function explicitly instead of using:
```suggestion
        super().__init__()
```
?

Reply:
Only because it is done this way in LinkParser, ~10 lines above. I can change both if super is preferred.

Reply:
I would change it. `super().__init__()` follows any refactor we might do later, without the possibility of leaving a wrong method call if we overlook that line.